lab/src/Microsoft.SyndicationFeed/src/Microsoft.SyndicationFeed.csproj

```diff
@@ -3,7 +3,15 @@
     <VersionPrefix>1.0.0</VersionPrefix>
     <VersionSuffix>preview</VersionSuffix>
```

Comment:
I know this is not part if this PR... I am wondering if this should be "preview1" instead. Will this be part of package version?